spec/views/research_sessions/preview.html.erb_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-describe 'research_sessions/preview' do
+xdescribe 'research_sessions/preview' do
```

Comment:
Should we just remove this?

Reply:
We can enable these tests when the new components for the preview are done. We should not need to change the tests (apart from maybe selectors) which will then validate the new components.